src/micro_http/src/response.rs

```diff
@@ -133,7 +133,9 @@ impl ResponseHeaders {
 /// Wrapper over an HTTP Response.
 ///
 /// The Response is created using a `Version` and a `StatusCode`. When creating a Response object,
-/// the body is initialized to `None`. The body can be updated with a call to `set_body`.
+/// the body is initialized to `None` and the header is initialized by `default`. The body can be
```

Comment:
Here `initialized by default` sounds a little bit weird. You can replace it with `initialized with the default value` or something similar.

Reply:
done 👍